session.go

```diff
@@ -33,40 +33,7 @@ func GetSessionOptions(c *Config) (*session.Options, error) {
 		return nil, err
 	}
 
-	// Call Get to check for credential provider. If nothing found, we'll get an
```

Comment:
Looking through the chain of events there appears to be an error check for the "NoCredentialProviders" error within the call to GetCredentials which then gets bubbled up here. So I removed this call for now and validated that with the current testing in place nothing breaks. This still needs some additional validation :man_shrugging: